docs/release-notes/release-notes-0.19.0.md

```diff
@@ -45,6 +46,12 @@
 ## Breaking Changes
 ## Performance Improvements
 
+## Misc
+
+* [Add retry logic for block fetching](https://github.com/lightningnetwork/lnd/pull/8381)
```

Comment:
please squash this commit with your main commit and remove the merge commit to have a clean commit history (you can just reset to master and then cherry pick your commit), ideally you'd end up with a single commit that contains all your changes

Reply:
Done